tests/test_diffeq/test_ode/test_ivp_examples.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+
+import numpy as np
+
+from probnum.diffeq.ode import ivp, ivp_examples
+from probnum.random_variables import Constant
+from tests.testing import NumpyAssertions
+
+
+class TestExamples(unittest.TestCase, NumpyAssertions):
+    """Test cases for example IVPs: Lotka-Volterra, etc."""
+
+    def setUp(self):
+        self.tspan = (0.0, 4.212)
+
+    def test_logistic(self):
```

Comment:
what do you thjink about a test of calling the ODE function rhs? This could either be in test_logistic or in another test, I dont care. Wouldnt this also remove lots of the code coverage complaint?

Reply:
It would. I'm going to take care of this

Reply:
Thanks a lot for this! In principle, I think one could refactor those tests out in the future, though for now, I like it as is.